aikau/src/main/resources/alfresco/lists/views/layouts/Grid.js

```diff
@@ -257,19 +257,22 @@ define(["dojo/_base/declare",
       resizeCell: function alfresco_lists_views_layouts_Grid__resizeCell(containerNodeMarginBox, widthToSet, node, /*jshint unused:false*/ index) {
          domStyle.set(node, {"width": widthToSet});
          var dimensions = {
-            w: widthToSet,
-            h: null
-         };
-         array.forEach(node.children, lang.hitch(this, "resizeWidget", dimensions));
+               w: widthToSet,
+               h: null
+            },
+            widgetsToResize = query("[widgetId]", node); // Resize all contained widgets, not just immediate children.
```

Comment:
Are we happy calling resize on every descendant widget, given that Dojo widgets often resize their children as well, when their own resize function is called? Is this definitely better than just finding the first descendant-generation with  resize functions on them (although obviously it's much more succinct)?